sds_data_manager/lambda_code/SDSCode/dynamodb_utils/processing_status.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class ProcessingStatus(Enum):
```

Comment:
```suggestion
class ProcessingStatus(IntEnum):
```
Do we want to use IntEnum? Probably doesn't make a huge difference here.

Reply:
not particularly